thirdeye/thirdeye-hadoop/pom.xml

```diff
@@ -151,6 +152,10 @@
                                 <relocation>
                                     <pattern>org.</pattern>
                                     <shadedPattern>thirdeye.org.</shadedPattern>
+                                    <excludes>
+                                        <exclude>org.apache.hadoop.**</exclude>
+                                        <exclude>org.apache.httpcomponents.**</exclude>
```

Comment:
The scope of that package is the same as Hadoop, which is "provided". So we will get runtime error when thirdeye-hadoop runs the code from httpcomponents if we don't exclude it.